elderly_assistant/workflow/reminder.py

```
@@ -226,8 +233,23 @@ def check_medication_trigger(now, poller, reminder_state, buzzer, display, logge
         matched_reminders = []
         for s in poller.schedules:
             t = _normalize_hhmm(s.get('time'))
-            if not t or t != now_hm:
+            if not t:
                 continue
+            # 精确到分钟的匹配：当前正处于该提醒的分钟
+            if t == now_hm:
+                pass
+            else:
+                # 补触发：计划时间已过去（当天）但未超过宽限窗口，避免轮询延迟漏提醒
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)